### PR TITLE
FAI-469 - Fix saliency response

### PR DESCRIPTION
**Pull request by r00ta**
https://issues.redhat.com/browse/FAI-469

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Review by manstis — APPROVED** [review]
LGTM.

I think we should have some form on integration test for these scenarios (vs list of changes.. as a list maintained by _people_ is likely to be flawed at some point so has lessor value than something we cannot cheat!).

**Comment by r00ta** [comment]
Jenkins retest this please

File: trusty/trusty-storage/trusty-storage-api/src/main/java/org/kie/kogito/trusty/storage/api/model/FeatureImportanceModel.java [hunk 1]
```diff
@@ -47,10 +47,10 @@ public void setFeatureName(String featureName) {
     }
 
     public Double getScore() {
-        return score;
+        return featureScore;
     }
 
     public void setScore(Double score) {
```
Comment (danielezonca) [inline trusty/trusty-storage/trusty-storage-api/src/main/java/org/kie/kogito/trusty/storage/api/model/FeatureImportanceModel.java]: Can you please align `get/set` name with `FeatureScore`?

**Review by danielezonca — APPROVED** [review]

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by r00ta** [comment]
Jenkins retest this please

**Pull request merged**